beamer/tests/contracts/test_l1_resolution.py

```diff
@@ -96,27 +94,27 @@ def test_restricted_calls(contracts, resolver, request_manager):
     # fill_manager -> messenger1 -> L1 resolver ->
     # messenger2 -> request manager
 
-    with brownie.reverts("RestrictedCalls: call disallowed"):
+    with ape.reverts("RestrictedCalls: call disallowed"):
         contracts.l2_messenger.sendMessage(resolver.address, b"")
 
-    with brownie.reverts("RestrictedCalls: call disallowed"):
-        contracts.resolver.resolve(
-            0, 0, brownie.chain.id, brownie.chain.id, caller, {"from": caller}
-        )
+    with ape.accounts.test_accounts.use_sender(caller):
 
-    with brownie.reverts("RestrictedCalls: call disallowed"):
-        contracts.resolver.resolve(
-            0, 0, brownie.chain.id, brownie.chain.id, ADDRESS_ZERO, {"from": caller}
-        )
+        with ape.reverts("RestrictedCalls: call disallowed"):
+            contracts.resolver.resolve(b"0", b"0", ape.chain.chain_id, ape.chain.chain_id, caller)
+
+        with ape.reverts("RestrictedCalls: call disallowed"):
+            contracts.resolver.resolve(
+                b"0", b"0", ape.chain.chain_id, ape.chain.chain_id, ADDRESS_ZERO
+            )
 
-    with brownie.reverts("RestrictedCalls: call disallowed"):
-        contracts.l1_messenger.sendMessage(request_manager.address, b"")
+        with ape.reverts("RestrictedCalls: call disallowed"):
+            contracts.l1_messenger.sendMessage(request_manager.address, b"")
 
-    with brownie.reverts("RestrictedCalls: call disallowed"):
-        contracts.request_manager.resolveRequest(0, 0, brownie.chain.id, caller, {"from": caller})
+        with ape.reverts("RestrictedCalls: call disallowed"):
+            contracts.request_manager.resolveRequest(b"0", b"0", ape.chain.chain_id, caller)
 
-    with brownie.reverts("RestrictedCalls: messenger not set"):
-        contracts.request_manager.resolveRequest(0, 0, 0, caller, {"from": caller})
+        with ape.reverts("RestrictedCalls: messenger not set"):
+            contracts.request_manager.resolveRequest(b"0", b"0", 0, caller)
```

Comment:
[Totally unimportant in this particular case, but wanted to mention...]
Question: how is `b"0"` actually represented?

Spoiler: what does `tuple(b"0")` evaluate to?  ;-)

Reply:
looked at this and it is 48 which is in ascii zero :D